docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc

```diff
@@ -10,24 +10,39 @@ ifdef::review[REVIEWERS: it would be great to provide some comments explaining t
 
 The blueprint declares which streamlets join together to form our pipeline. Create `blueprint.conf` in `src/main/blueprint` to specify the connection as follows:
 
-```
-blueprint {
-  streamlets {
-    http-ingress = sensordata.SensorDataHttpIngress
-    metrics = sensordata.SensorDataToMetrics
-    validation = sensordata.MetricsValidation
-    valid-logger = sensordata.ValidMetricLogger
-    invalid-logger = sensordata.InvalidMetricLogger
-  }
-
-  connections {
-    http-ingress.out = [metrics.in]
-    metrics.out = [validation.in]
-    validation.invalid = [invalid-logger.in]
-    validation.valid = [valid-logger.in]
-  }
-}
-```
+[source,json]
+----
+include::{cloudflow-examples-version}@docsnippets:ROOT:example$sensor-data-scala/src/main/blueprint/blueprint.conf[tag=all]
+----
+
+
+== Understanding the Blueprint File.
+The blueprint file contains two sections: the `streamlets` section and the `topics` section.
+
+=== The `streamlets` section
+In the `streamlets` section of the blueprint, we declare the streamlets participating in the pipeline definition. 
```

Comment:
Can we use `application` instead of `pipeline definition`?

Reply:
we could. But "application" is everywhere. This is a bit more specific.